packages/contract/lib/statuserror.js

```diff
@@ -11,7 +11,9 @@ class StatusError extends TruffleError {
 
     if (reason) reasonString = `Reason given: ${reason}.`;
 
-    if (utils.bigNumberify(receipt.gasUsed).eq(utils.bigNumberify(gasLimit))) {
+    if (
+      utils.BigNumber.from(receipt.gasUsed).eq(utils.BigNumber.from(gasLimit))
```

Comment:
I feel like our prettier config could do a bit better in cases like this 🤔 

not suggesting that fixing that should be in scope for this review, however